tools/split.py

```diff
@@ -48,6 +48,18 @@ def make_cut(video, start, end, outfile):
     ])
     return ret == 0
 
+def make_thumbnail(video, time, outfile):
+    ret = subprocess.call([
+        "ffmpeg",
+        "-y",               # Yes, overwrite
+        "-ss", str(time),   # Time of thumbnail capture
+        "-i", video,        # Input
+        "-frames", "1",     # Number of frames
+        outfile,            # Output
+    ])
```

Comment:
ffmpeg emitted this complaint:
```
[image2 @ 0x556dad9a7ac0] The specified filename 'workdir/Carmel-Thursday/Event-Driven-Applications-on-Kubernetes.png' does not contain an image sequence pattern or a pattern is invalid.
[image2 @ 0x556dad9a7ac0] Use a pattern such as %03d for an image sequence or use the -update option (with -frames:v 1 if needed) to write a single image.
```
It still spit out an image though, so not a big concern.

Reply:
I didn't get this error when using ffmpeg, so not sure if we need to address it immediately.